tests/test_FindTest.py

```diff
@@ -181,8 +181,11 @@ def test_find_citations(self):
             # Test with page range with a weird suffix
             ('559 N.W.2d 826|N.D.',
              [case_citation(page='826', reporter='N.W.2d', volume='559')]),
-            # Test with malformed/missing page number
+            # Test with malformed page number
             ('1 U.S. f24601', []),
+            # Test with page number that is indicated as missing
+            ('1 U.S. ___',
+             [case_citation(volume='1', reporter='U.S.', page='___')]),
```

Comment:
Do we use None for the `page` attribute in any other context? Maybe it's better to normalize these underscore things to `None` and then that answers your question about whether we need to add an extra attribute to the response to indicate this situation. What do you think?

Reply:
That's a good idea, I don't think we use `None` in the `page` attribute at all yet!

Reply:
Fixed in a120ca8